REQUIRE

```diff
@@ -1,3 +1 @@
-julia 0.6
-Compat 0.65.0
-Nullables 0.0.3
+julia 1.0
```

Comment:
Can we get rid of the REQUIRE file completely now? I don't think anything uses it anymore

Reply:
Yes, I think you are correct in that this file can now be deleted. I haven't gone through the new Registrator process yet so I was being cautious.